griptape/structures/structure.py

```diff
@@ -63,6 +65,10 @@ class Structure(ABC):
     )
     meta_memory: MetaMemory = field(default=Factory(lambda: MetaMemory()), kw_only=True)
     fail_fast: bool = field(default=True, kw_only=True)
+    futures_executor_fn: Callable[[], futures.Executor] = field(
+        default=Factory(lambda: lambda: futures.ThreadPoolExecutor()), kw_only=True
```

Comment:
Double λ takes a couple seconds to digest

Reply:
[original implementation here](https://github.com/griptape-ai/griptape/pull/910)

Reply:
"Yo, dawg..."